source/virtualBuffers/gecko_ia2.py

```diff
@@ -552,7 +552,9 @@ def _searchableAttribsForNodeType(self,nodeType):
 		elif nodeType == "figure":
 			attrs = [
 				{"Iaccessible::role": [oleacc.ROLE_SYSTEM_GROUPING]},
-				{"IAccessible2::attribute_xml-roles": [VBufStorage_findMatch_word("figure")]}
+				{"IAccessible2::attribute_xml-roles": [VBufStorage_findMatch_word("figure")]},
+				# Needed so that navigation by figure works for HTML figures in Chromium
+				{"IAccessible2::attribute_tag": self._searchableTagValues(["figure"])}

```

Comment:
Terminating lists with commas minimize diffs like this where you add an item

```suggestion
				{"IAccessible2::attribute_tag": self._searchableTagValues(["figure"])},
```